pkg/instance/executor.go

```diff
@@ -37,11 +39,11 @@ func NewExecutor(ctx context.Context, sysDeps system.SystemDependencies) (*Execu
 		return nil, ErrSettingArgs.Wrap(err)
 	}
 
-	if err := i.SetMemory(memoryLimit, memoryLimit); err != nil {
+	if err := i.SetMemory(resource.MustParse(memoryLimit), resource.MustParse(memoryLimit)); err != nil {
```

Comment:
We should validate these at compile by moving `memoryLimit` and also `cpuLimit` from `const` to `var`

```go
var (
	memoryLimit = resource.MustParse("100M")
	cpuLimit    = resource.MustParse("100m")
)
```